classes/class-wc-connect-shipping-label.php

```diff
@@ -238,6 +238,7 @@ public function meta_box( $post ) {
 				'purchaseURL'             => get_rest_url( null, '/wc/v1/connect/shipping-label' ),
 				'addressNormalizationURL' => get_rest_url( null, '/wc/v1/connect/normalize-address' ),
 				'getRatesURL'             => get_rest_url( null, '/wc/v1/connect/shipping-rates' ),
+				'labelPreviewURL'         => plugins_url( 'assets/fake_label.png', dirname( __FILE__ ) ),
```

Comment:
We will probably need to have labels for other carriers soon, so let's use the carrier name in the filename to make this easier to distinguish between other carrier's labels, e.g. `usps_sample_label.png`